src/Facade/ResourceBuilders/PurchaseOrdersResourceBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Linn.Purchasing.Facade.ResourceBuilders
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Linn.Common.Facade;
+    using Linn.Purchasing.Domain.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps.PartSuppliers;
+    using Linn.Purchasing.Resources;
+
+    public class PurchaseOrdersResourceBuilder : IBuilder<IEnumerable<PurchaseOrder>>
```

Comment:
as per my post on Teams, these IBuilder<IEnumerable<...>>'s are redundant now. Can delete this class

Reply:
Oh, good catch. Will do.